### PR TITLE
Add tests for AddMusiciantoBandCommand

### PR DESCRIPTION
**Pull request by DeltonCheng**
This commit adds the unit tests for AddMusiciantoBandCommand to check the outputs and the pathways of the methods in the class.

### CODE REVIEW

**Comment by DeltonCheng** [comment]
I had to add get(Index) functions for UniqueBandList and UniqueMusicianList due to the changes @yytan25 had made to the AddMusiciantoBandCommand when implementing 1-indexing (using FilteredList).

@yytan25 please do help take a look at the pathway of the lastShownBandList/MusicianList in AddMusiciantoBand and compare it to the tests. Thanks!

File: src/main/java/seedu/address/logic/commands/AddMusiciantoBandCommand.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import seedu.address.model.musician.Musician;
 
 /**
- * Adds a musician to a band.
+ * Adds a musician to a band.j
```
Comment (yytan25) [inline src/main/java/seedu/address/logic/commands/AddMusiciantoBandCommand.java]: Nit: remove extra character

**Review by yytan25 — APPROVED** [review]
LGTM, apart from the small typo in the Javadoc of `AddMusicianToBandCommand`. Thanks for adding the necessary functions and for the detailed test cases!

Please correct the typo before merging the branch!

File: src/main/java/seedu/address/logic/commands/AddMusiciantoBandCommand.java (hunk 1 above)
Comment (DeltonCheng) [reply src/main/java/seedu/address/logic/commands/AddMusiciantoBandCommand.java]: Have rectified this typo, thank you!

**Pull request merged**